src/app/compiler/angular/deps/helpers/component-helper.ts

```diff
@@ -135,6 +135,21 @@ export class ComponentHelper {
         return this.symbolHelper.getSymbolDeps(props, 'inputs', srcFile);
     }
 
+    public getInputSignals(props) {
+        let inputSignals = [];
+        props?.forEach((prop, i) => {
+            const regexp = /input(?:\.(required))?(?:<([\w-]+)>)?\(([\w-]+)?\)/;
```

Comment:
@astrutz This regex has quite some limitations. I will list the most obvious ones and potentially we can adjust the regex to cover the cases:

- `input("helloworld")` -> strings are not supported as default values
- `input(false, { transform: booleanAttribute });` -> as soon as a transformation option is used, the regex doesn't match

I am thinking of potentially rather use some babel parsing here instead of relying on a regex, since the expression might also have some new line characters, especially if the options object is used to provide a transform function

Reply:
Related: https://github.com/storybookjs/storybook/pull/26413#issuecomment-2073409676

Reply:
@vogloblinsky thank you so much for this if you can support the cases @valentinpalkovic mentioned, thank you so much in advance.

Reply:
@valentinpalkovic I agree with you, although I can't really do much, as I don't have the capacity atm. I'll check with company and can come back to you in about 3 weeks

Reply:
Is there a Github issue ticket for that?